Tests/ViewInspectorTests/SwiftUI/TreeViewTests.swift

```diff
@@ -28,6 +28,16 @@ final class TreeViewTests: XCTestCase {
         }
         XCTAssertEqual(sut.content.medium.viewModifiers.count, count)
     }
+
+  func testVariadicViewTree() throws {
```

Comment:
Nit: we probably want to use 4 spaces of indentation per tab in this repo.

Reply:
Thanks for catching this! Just pushed a fix to use 4 spaces.